docs/docs/guides/customize-identity-model.mdx

```diff
@@ -0,0 +1,304 @@
+Currently, only the `email` method is available (recovery/verification via a link
+sent in an email).
+
+```diff
+    "traits": {
+      "type": "object",
+      "properties": {
++       "username": {
++         "type": "string",
++         "title": "Username",
++         "ory.sh/kratos": {
++           "credentials": {
++             "password": {
++               "identifier": true
++             }
++           },
+            "recovery": {
+              "via": "email"
+            },
+            "verification": {
+              "via": "email"
+            }
+          }
+        }
+      },
+      "required": [
+        "username"
+      ],
+      "additionalProperties": false
+    }
+```
+
+To add traits to the Identity Model, add them inside the `traits "properties"`
+
+```json
+"traits": {
+"type": "object",
+"properties": {
+// your identities traits go here
+       }
+    }
+```
+
+for example the GitHub Handle as `string` :
+
+```json
+"traits": {
+"type": "object",
+"properties": {
+        "handle": {
+          "type": "string",
+          "title": "Your GitHub Handle"
+        }
+      }
+    }
+```
+
+or a mobile phone number as `integer`:
+
+```json
+"traits": {
+"type": "object",
+"properties": {
+        "phone-number" : {
+            "type": "integer",
+            "title" : "Your Mobile Phone Number"
+        }
+      }
+    }
+```
+
+Possible values for the `type` are `string`, `number`, `integer`, `boolean`.
+
+Use `string` for text fields, `boolean` for true/false fields, `integer` or
+`number` for integral or floating-point numbers. If you want to know more about
+these types, please refer to the
+[json-schema documentation](https://json-schema.org/understanding-json-schema/reference/type.html).
+
+The `title` of each field is what the user as description or sample input. After
+adding the above examples the sign-up screen would look like so:
+
+![Example Identity Model with GitHub and Phone Number Fields](../images/identity-model/example-identity0.png)
+
+### Required traits
+
+To define certain traits as required, add them to the `required` array by name:
+
+```json
+  "properties": {
+    "traits": {
+      "type": "object",
+      "properties": {
+// your identities traits go here
+       },
+      "required": [
+/add required fields to this array
+        "email"
+      ],
+    }
+  }
+```
+
+### Additional Properties
+
+The
+[`additionalProperties`](https://json-schema.org/understanding-json-schema/reference/object.html#additional-properties)
+keyword is used to control the handling of properties whose names are not listed
+in the properties keyword. This usually should be set to false.
+
+```json
+      "additionalProperties": false
+```
+
+## Example Identity Model
+
+The following Identity Model includes first/last and nickname, as well as number
+fields for the phone number and birthday. There are also two true/false fields
+for specifying the newsletter subscription and enterprise status.
```

Comment:
integer for phone number isn't ideal

Reply:
I will change it to `age` 🙂 

Do you think it makes sense to include a notice that phone numbers are not really supported right now?